tests/src/test/scala/whisk/core/cli/test/WskApiGwTests.scala

```diff
@@ -42,6 +42,7 @@ class WskApiGwTests extends BaseApiGwTests with WskActorSystem with JsHelpers wi
   val wsk: common.Wsk = new Wsk
 
   it should "reject apimgmt actions that are invoked with not enough parameters" in {
+    val x = true
```

Comment:
not sure where `x` is used

Reply:
not sure where that came from! removed. thanks.